docs/packages/Rule_Tester.mdx

```diff
@@ -233,6 +233,28 @@ RuleTester.itOnly = vitest.it.only;
 RuleTester.describe = vitest.describe;
 ```
 
+#### Node built-in test runner
+
+Consider setting up `RuleTester`'s static properties in a preloaded module using the [`--import`](https://nodejs.org/api/cli.html#--importmodule) or [`--require`](https://nodejs.org/api/cli.html#-r---require-module) flag:
+
+```ts
+// setup.js
+import * as test from 'node:test';
+import { RuleTester } from '@typescript-eslint/rule-tester';
+
+RuleTester.afterAll = test.afterAll;
+
+RuleTester.it = test.it;
+RuleTester.itOnly = test.it.only;
+RuleTester.describe = test.describe;
```

Comment:
Nit: any reason for the blank line?

```suggestion
RuleTester.describe = test.describe;
RuleTester.it = test.it;
RuleTester.itOnly = test.it.only;
```

Reply:
I was trying to mirror the above example (which admittedly has a comment too)